pyproject.toml

```diff
@@ -1,14 +1,17 @@
 [tool.poetry]
-name = "project-template"
-version = "0.1.0"
-description = "Simple Python project template"
-authors = [""]
-license = "MIT"
+name = "CASTanet"
+version = "1.0.0"
+description = "Program to identify different syntax in Python programs"
+authors = ["Thomas Antle, Caden Hinckley, Maddy Kapfhammer, Bailey Matrascia, Nolan Thompson"]
+license = "AC"
+
+[tool.poetry.scripts]
+CASTanet = "CASTanet.main:app"
```

Comment:
It's generally good practice to keep executables all lowercase. Additionally, your main package is `castanet`, not `CASTanet`. Finally, `main.py`, which I think is what you're attempting to call here, is not in the `castanet` package.

Reply:
This does not work as-is. This should be `castanet = "main:app"`.

Reply:
This has been resolved, closing conversation!